include/algorithms/decision_forest/decision_forest_regression_training_types.h

```diff
@@ -109,10 +109,27 @@ enum ResultEngineId
     lastResultEngineId = updatedEngine
```

Comment:
Do not forget to update the parameter as a method. So that all declarations of the parameter field go to the side of compiling the library to support further BC.

Reply:
It was added